lint-configs/pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+line-length = 145
+target-version = ['py39']
+verbose = true
+
+[tool.isort]
+line_length = 145
```

Comment:
145 line length is quite a lot. I am used to 79 or 88, which are also better lengths in my opinion. However, I am aware that the first PR for each connector after changing the line-lenght will be a nightmare to review.

Reply:
I'm partial to 120 as I feel 79 and 88 are way too short, but in this case this is dictated by what we already have for flake and isort.

Reply:
Ok

Reply:
So it's probably "all in" for all hooks or leave it as it is

Reply:
Ahhh the neverending grand battle for line lenghts ;) 
I am also on the team 79/88, but I also agree, in this case, when migrating on the existing projects, this may create ton of changes that will cover actual valuable commit messages in the history. And in such case, I prefer to stay with what's been set up already - this 145 it is for me.

Reply:
+1 for 145

Reply:
At least for now 145 it is